assets/js/components/notifications/ErrorNotifications.js

```diff
@@ -24,16 +24,24 @@ import { Fragment } from '@wordpress/element';
 /**
  * Internal dependencies
  */
+import Data from 'googlesitekit-data';
+import { CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+import Notification from '../legacy-notifications/notification';
 import AuthError from './AuthError';
 import UnsatisfiedScopesAlert from './UnsatisfiedScopesAlert';
-import InternalServerError from './InternalServerError';
+const { useSelect } = Data;
 
 export default function ErrorNotifications() {
+	const internalServerError = useSelect( ( select ) => select( CORE_SITE ).getInternalServerError() );
+
 	return (
 		<Fragment>
+			{ internalServerError && (
+				<Notification { ...internalServerError } />
+			) }
```

Comment:
Please restore the `InternalServerError` component as per the ACs. It is a very simple component but let's keep it independent for now as defined.

Reply:
Restored.